src/users/store/actions/user.actions.ts

```diff
@@ -122,6 +125,21 @@ export class ReinvitePendingUser {
   constructor(public payload: any) {}
 }
 
+export class LoadAccessTypes {
+  public readonly type = LOAD_ACCESS_TYPES;
+  constructor(public payload?: string[]) {}
+}
+
+export class LoadAccessTypesSuccess implements Action {
+  public readonly type = LOAD_ACCESS_TYPES_SUCCESS;
+  constructor(public payload: any) {} 
```

Comment:
The `LoadAccessTypesSuccess` payload should be of type `JurisdictionAccessType[]`
```suggestion
  constructor(public payload: JurisdictionAccessType[]) {} 
```